docs/SETUP.md

```diff
@@ -81,3 +81,14 @@ TODO
 
 # Setup Python-Flask-Postgres Backend
 TODO
+1. Create the database
+    ```
+    $ psql
```

Comment:
I think this needs to be `psql <database-name>` like `psql postgres` but in our case `psql startercode`. Do these instructions work on your machine?
It would be best to have sql alchemy create the db for us. Second best thing would be to have a `.sh` script

Reply:
yes they work